GVFS/GVFS.Common/GVFSPlatform.cs

```diff
@@ -138,16 +138,19 @@ public class UnderConstructionFlags
             public UnderConstructionFlags(
                 bool supportsGVFSUpgrade = true,
                 bool supportsGVFSConfig = true,
-                bool requiresDeprecatedGitHooksLoader = false)
+                bool requiresDeprecatedGitHooksLoader = false,
+                bool supportsNuGetEncryption = true)
             {
                 this.SupportsGVFSUpgrade = supportsGVFSUpgrade;
                 this.SupportsGVFSConfig = supportsGVFSConfig;
                 this.RequiresDeprecatedGitHooksLoader = requiresDeprecatedGitHooksLoader;
+                this.SupportsNuGetEncryption = supportsNuGetEncryption;
             }
 
             public bool SupportsGVFSUpgrade { get; }
             public bool SupportsGVFSConfig { get; }
             public bool RequiresDeprecatedGitHooksLoader { get; }
+            public bool SupportsNuGetEncryption { get; }
```

Comment:
As discussed offline, `UnderConstructionFlags` has been for flags where we have remaining work on our side for some platforms.  If we're waiting on NuGet to support encryption the flag/property could be moved out of `UnderConstructionFlags`.